config/environments/development.rb

```diff
@@ -51,7 +51,7 @@
   }
 
   # Raises error for missing translations
-  config.action_view.raise_on_missing_translations = true
+  # config.action_view.raise_on_missing_translations = true
```

Comment:
We probably want to keep this.

Reply:
We don't, I told her to get rid of it. That setting really breaks the transifex workflow, which is supposed to be:

1. Add translations to your primary language
2. Push those translations to the branch
3. Update the additional languages in transifex
4. Download the new transifex files for the non-primary languages and merge them into the project

That breaks down if the app crashes in development every time you don't immediately manually translate all of the fields, and it makes it so we can't offload the translation work to people from the client's org/team.